src/components/Card.js

```diff
@@ -31,51 +67,114 @@ const Wrapper = styled.div`
   padding: 15px;
   border-radius: 13px;
   box-shadow: 0px 0px 20px #dedede;
+  display: flex;
+  flex-direction: column;
+`;
+
+const Heading = styled.div`
+  display: grid;
+  grid-auto-columns: 1fr;
+  grid-template-columns: 0.2fr 1.4fr 1.4fr;
 `;
 
 const Icon = styled.img`
   height: 18px;
   width: auto;
   margin-top: 14px;
+  grid-column: 1 / 2;
 `;
 
-const CardTitle = styled.div`
-  display: flex;
-  justify-content: flex-start;
-`;
-
-const Spacer = styled.div`
-  width: 10px;
+const Title = styled.p`
```

Comment:
I would expect `Title` to be a heading element! Maybe a `h2`?

Reply:
Understood, I changed it to "Name".